src/components/IncidentListItem.vue

```diff
@@ -61,7 +71,7 @@ export default {
 
   methods: {
     mouseUp(event) {
-      if (event.target === this.$refs.link.$el) return
```

Comment:
This was previously buggy, causing multiple navigations if you clicked on the anchor element